packages/edit-post/src/components/layout/index.js

```diff
@@ -287,6 +288,57 @@ function Layout( { initialPost } ) {
 		);
 	}
 
+	const { createSuccessNotice } = useDispatch( noticesStore );
+
+	const onActionPerformed = useCallback(
```

Comment:
This callback is specific per editor at the moment. So I moved it into a prop to the "Sidebar" component. Ideally this should be the same between editors to be honest.